test/benchmarks/mongoBench/runner.js

```diff
@@ -39,7 +39,7 @@ function calculateMicroBench(benchmark, data) {
   const rawData = data.rawData;
   const count = data.count;
 
-  const sortedData = [].concat(rawData).sort();
+  const sortedData = [].concat(rawData).sort((a, b) => a - b);
```

Comment:
One last fix, can we calculate the median properly? average of middle two if even and the middle value if odd. (we could delete the other percentiles here, we don't use them)

Reply:
https://github.com/mongodb/specifications/blob/master/source/benchmarking/benchmarking.md#:~:text=For%20each%20micro,for%20every%20driver.